IonDotnet.Tests/Builders/LoaderTest.cs

```diff
@@ -46,19 +56,25 @@ public void TextLoader_SymbolAnnotation()
             const string doc = "$3::123";
             var datagram = IonLoader.Default.Load(doc);
 
-            var child = (IonInt) datagram[0];
-            var annots = child.GetTypeAnnotations();
-            Assert.AreEqual(1, annots.Count);
-            Assert.AreEqual(SystemSymbols.IonSymbolTable, annots.First().Text);
+            foreach (var ionValue in datagram)
+            {
+                var child = (IonInt)ionValue;
+                var annots = child.GetTypeAnnotations();
+                Assert.AreEqual(1, annots.Count);
+                Assert.AreEqual(SystemSymbols.IonSymbolTable, annots.First().Text);
+            }
         }
 
         [TestMethod]
         public void TextLoader_TripleQuotedClob()
         {
             const string doc = "{{'''hello'''}}";
             var datagram = IonLoader.Default.Load(doc);
-            var child = (IonClob) datagram[0];
-            Assert.AreEqual("hello".Length, child.Bytes().Length);
+            foreach (var ionValue in datagram)
```

Comment:
There should only be one element, so I'd assert that there's only one and then index element zero instead of looping.

Reply:
We cannot access elements via indexing since ``IIonValue`` is an interface, hence looping. We get ``Cannot apply indexing to type IIonValue `` error.

Added assertion for number of elements.